### PR TITLE
Add ip right policies for PrestaShop open source user groups

### PR DESCRIPTION
**Pull request by ttoine**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | see title?
| Type?         | new content
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | just read.

### CODE REVIEW

File: user-groups/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Step 2, use the templates
+
+When you have read the guidelines and you are ready to follow them, you can then download the [ZIP archive with all the templates](PrestaShop_Privacy_Policies.zip).
+
+You will find in it:
+- A copy of the guidelines in PDF, for reference
+- The Montserrat-Bold.ttf text font, used for the logo
+- Horizontal user group logo template, one line, EPS
+- Horizontal user group logo template, two linew, EPS
```
Comment (sowbiba) [inline user-groups/readme.md:21]: lines

File: user-groups/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- User group single logo, SVG
+- User group sigle logo, EPS
```
Comment (sowbiba) [inline user-groups/readme.md:23]: single

File: user-groups/readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Vertical user group logo template, one line, EPS
+- Vertical user group logo template, two lines, EPS
+
+In order to edit them, you will need a vector graphic software like Inkscape (free software) or Adobe Illustrator, or an equivalent.
+
+
+## Tell us more about your user group
+
+When you have created you user group logo and published it on your forum, group, chat, or your favourite communication channel, please tell the world about it: add it to the list in the dedicated document (coming soon).
```
Comment (sowbiba) [inline user-groups/readme.md:32]: favorite

**Review by sowbiba — CHANGES_REQUESTED** [review]

File: user-groups/readme.md (hunk 1 above)
Comment (ttoine) [reply user-groups/readme.md]: 👍

File: user-groups/readme.md (hunk 2 above)
Comment (ttoine) [reply user-groups/readme.md]: 👍

File: user-groups/readme.md (hunk 3 above)
Comment (ttoine) [reply user-groups/readme.md:32]: 👍

**Pull request merged**